samples/sandbox/src/index.html

```diff
@@ -5,7 +5,9 @@
     <link rel='stylesheet' type='text/css' href='sample.css'>
   </head>
   <body>
-    If you see this then something is wrong.
+    <div id="main">
+      If you see this then something is wrongly.
```

Comment:
can you fix this in this PR? otherwise when I merge this it's going to end up in the file